virttest/utils_misc.py

```diff
@@ -196,6 +196,23 @@ def get_usable_memory_size(align=None):
     return usable_mem
 
 
+def align_value(value_to_be_aligned, align=1024):
```

Comment:
parameter align rename to align_factor to be more meaningful

Reply:
`align_value(value, factor=1024)`

Reply:
@chunfuwen Sure, will change.